instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/Injection.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet;
+
+import io.opentelemetry.instrumentation.api.util.VirtualField;
+import javax.servlet.ServletOutputStream;
+
+public class Injection {
+
```

Comment:
private static final for VirtualField.find(ServletOutputStream.class, InjectionState.class) and use below